app/src/main/java/com/example/fbufinalapp/DashboardFragment.java

```diff
@@ -149,6 +150,8 @@ public void fetchTimelineAsync(int page) {
     private void queryPosts() {
         ParseQuery<Itinerary> query = ParseQuery.getQuery(Itinerary.class);
         query.include(Itinerary.KEY_USER);
+        query.whereEqualTo("authors", ParseUser.getCurrentUser());
```

Comment:
Is there a place where the keys for a user is kept? 
If so, then it is best to, similar to reference that code to have a centralized place for all the ParseUser fields. 
If not,  it's a good idea to have a class enum for ParseUsers that has all the fields. Something like this  :

```
class User {
ID = "id"
NAME = "name"
AUTHORS = "authors" 
}
```

This document would explain how best to create an enum type to do something along those lines in Java. 
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

Reply:
yes! I can add these to a central file :)